src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -30,32 +27,28 @@ public static String getAddCommand(Person person) {
      */
     public static String getPersonDetails(Person person) {
         StringBuilder sb = new StringBuilder();
-        sb.append(PREFIX_NAME + person.getName().name + " ");
-        sb.append(PREFIX_PHONE + person.getPhone().phone + " ");
-        sb.append(PREFIX_EMAIL + person.getEmail().email + " ");
-        sb.append(PREFIX_ADDRESS + person.getAddress().address + " ");
+        sb.append(PREFIX_NAME).append(person.getName().toString()).append(" ");
+        sb.append(PREFIX_PHONE).append(person.getPhone().toString()).append(" ");
+        sb.append(PREFIX_EMAIL).append(person.getEmail().toString()).append(" ");
+        sb.append(PREFIX_ADDRESS).append(person.getAddress().toString()).append(" ");
```

Comment:
Should these be changed? Would "sb.append(PREFIX_NAME + person.getName() + " ");" work instead?

Reply:
Yes, it is recommended by IntelliJ as `StringBuilder.append()` is more efficient than the `String` `+` operator.